src/main/java/firebase/sdk/dashboard/dao/SDKDaoDatastore.java

```diff
@@ -0,0 +1,343 @@
+
+    if (results == null) {
+      return Arrays.asList();
+    }
+
+    ArrayList<String> sdksForPlatform = new ArrayList<>();
+    for (Entity entity: results) {
+      sdksForPlatform.add(
+          (String) entity.getProperty("libraryName"));
+    }
+
+    return sdksForPlatform;
+  }
+
+  public List<String> getSDKsEnrolledInRelease(Platform platform, String releaseName) {
+    FilterPredicate platformPropertyFilter = makePropertyFilter("platform", platform.getValue());
+    FilterPredicate releasePropertyFilter = makePropertyFilter("releaseName", releaseName);
+    CompositeFilter compositeFilter = new CompositeFilter(
+        CompositeFilterOperator.AND, Arrays.asList(
+          platformPropertyFilter,
+          releasePropertyFilter));
+
+    Query query = new Query("SDKReleaseMetadata")
+      .setFilter(compositeFilter);
+
+    PreparedQuery preparedQuery = DATASTORE.prepare(query);
+    FetchOptions fetchOptions = FetchOptions.Builder.withDefaults();
+    QueryResultIterable<Entity> results = preparedQuery.asQueryResultIterable(fetchOptions);
+
+    if (results == null) {
+      return Arrays.asList();
+    }
+
+    ArrayList<String> enrolledSDKs = new ArrayList<>();
+    for (Entity entity: results) {
+      String libraryName = (String) entity.getProperty("libraryName");
+      enrolledSDKs.add(libraryName);
+    }
+
+    return enrolledSDKs;
+  }
+
+  public SDKReleaseMetadata getSDKReleaseMetadata(Platform givenPlatform, String givenReleaseName, String givenLibraryName) {
+    Key sdkReleaseMetadataKey = KeyFactory.createKey("SDKReleaseMetadata", givenPlatform.toString() + "_" + givenLibraryName + "_" + givenReleaseName);
+    FilterPredicate keyPropertyFilter = makePropertyFilter("__key__", sdkReleaseMetadataKey);
+
+    Query query = new Query("SDKReleaseMetadata")
+      .setFilter(keyPropertyFilter);
+
+    PreparedQuery preparedQuery = DATASTORE.prepare(query);
+    Entity entity = preparedQuery.asSingleEntity();
+
+    if (entity == null) {
+      return null;
+    }
+
+    Platform platform = Platform.fromInteger(((Number) entity.getProperty("platform")).intValue());
+    String libraryName = (String) entity.getProperty("libraryName");
+    String releaseName = (String) entity.getProperty("releaseName");
+    String releaseVersion = (String) entity.getProperty("releaseVersion");
+    String oldVersion = (String) entity.getProperty("oldVersion");
+    HashMap<String, String> additionalInfo = getAdditionalInfoFromProperty(
+        (EmbeddedEntity) entity.getProperty("additionalInfo"));
+
+    SDKReleaseMetadata sdkReleaseMetadata = SDKReleaseMetadata.newBuilder()
+      .platform(platform)
+      .libraryName(libraryName)
+      .releaseName(releaseName)
+      .releaseVersion(releaseVersion)
+      .oldVersion(oldVersion)
+      .additionalInfo(additionalInfo)
+      .build();
+
+    return sdkReleaseMetadata;
+  }
+
+  //TODO: Define exception
```

Comment:
Whats this for? Since you have implemented the method, can you please add any required descriptions in the same PR.